low-level/framework/sspl_ll_d

```diff
@@ -722,6 +723,7 @@ def set_log_level(signal_number, frame):
         log_level = log_level.decode('utf-8')
     logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
     logger.setLevel(log_level)
+    # Coverage report commands to stop, save and generate coverage report
```

Comment:
DO NOT EDIT: Marker comment to dynamically add code to stop coverage, save and generate code coverage report

Reply:
addressed